<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #4C4C4C</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="click_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: #E5D76B ; background-color: #2D2D2D; border: none;}
QPushButton:hover {color: #EBD84C ; background-color: #3A3A39; border: none;}
QPushButton:pressed {color: #FFE943 ; background-color: #474746; border: none;}</string>
    </property>
    <property name="text">
     <string>Click!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upgrade_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: #ff0000 ; background-color: #2D2D2D; border: none;}</string>
    </property>
    <property name="text">
     <string>Upgrade</string>
    </property>
   </widget>
   <widget class="QLabel" name="clicks">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: #37B8B6 ; background-color: #2D2D2D; border: none;}
QPushButton:hover {color: #42DCDA ; background-color: #3A3A39; border: none;}
QPushButton:pressed {color: #47F5F2 ; background-color: #474746; border: none;}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: #37B8B6 ; background-color: #2D2D2D; border: none;}
QPushButton:hover {color: #42DCDA ; background-color: #3A3A39; border: none;}
QPushButton:pressed {color: #47F5F2 ; background-color: #474746; border: none;}</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
